prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,61 @@
+function isAdult(age) {
+  return age < 18 ? false : true;
+}
```

Comment:
A ternary isn't even necessary here actually! Anytime you ever use a comparison operator, it will always evaluate to `true` or `false`. So you could just do `return age < 18` and it will already return true or false

Reply:
Wow I didn't think of that